website/conferences/views.py

```diff
@@ -215,6 +215,14 @@ def conference_view(**kwargs):
         )
         projects = Node.find(query)
         submissions = projects.count()
+        downloads_list = [
```

Comment:
This only influence the main meeting page which has all the meetings. It
doesn't change the behavior when user looks into posters or presentation of
each meeting.

On Wed, Jun 3, 2015 at 2:10 PM, Brian J. Geiger <notifications@github.com>
wrote:

> In website/conferences/views.py
> <https://github.com/CenterForOpenScience/osf.io/pull/2916#discussion_r31652096>
> :
>
> > @@ -215,6 +215,14 @@ def conference_view(**kwargs):
> >          )
> >          projects = Node.find(query)
> >          submissions = projects.count()
> > +        downloads_list = [
>
> So more popular conferences are more likely to have more projects and also
> more people looking at them, yes? If we tie up a worker for a few seconds
> each time on a big conference, it seems like the wrong confluence of
> popularity and slowness. Might want to think about a caching method or some
> kind of async way of doing this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/2916/files#r31652096>
> .
>